libevm/CMakeLists.txt

```diff
@@ -19,13 +19,11 @@ file(GLOB HEADERS "*.h")
 
 add_library(evm ${SOURCES} ${HEADERS})
 
-find_package(Eth)
-
-target_link_libraries(evm ${Eth_ETHCORE_LIBRARIES})
-target_link_libraries(evm ${Eth_EVMCORE_LIBRARIES})
-target_include_directories(evm PRIVATE .. ${JSONCPP_INCLUDE_DIR})
+# TODO: We have horrible cilcular dependency: evm depends also on ethereum
+#       library. Here we only explicitly add jsoncpp to fix includes.
```

Comment:
Instead of `${Eth_ETHCORE_LIBRARIES}` I'm using `ethcore`. Much simpler and we don't depend on external helper functions/macros. This is what everyone with basic CMake skill will understand. Knowing where  `${Eth_ETHCORE_LIBRARIES}` comes from is much harder.